Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_cff.py

```diff
@@ -15,7 +29,9 @@
 # DT calibration needs as many muons with DIGIs as possible, which in cosmic ray runs means standAloneMuons
 #    nHitMinGB = 1, # muon collections now merge globalMuons, standAlone, and trackerMuons: this stream has always assumed globalMuons
     ptMin=cms.double(0.),
-    pMin=cms.double(10.)
+    pMin=cms.double(25.)
```

Comment:
@anthonykrose a  "," is missing here, therefore causing compilation errors. Please add it.